controller/space_test.go

```diff
@@ -395,20 +395,20 @@ func (rest *TestSpaceREST) TestShowSpaceNotModifiedUsingIfNoneMatchHeader() {
 	assert.Equal(t, *created.Data.Attributes.Version, *fetched.Data.Attributes.Version)
 
 	// verify list-WI URL exists in Relationships.Links
-	require.NotNil(t, *fetched.Data.Relationships.Workitems)
-	require.NotNil(t, *fetched.Data.Relationships.Workitems.Links)
-	require.NotNil(t, *fetched.Data.Relationships.Workitems.Links.Related)
+	require.NotNil(t, fetched.Data.Relationships.Workitems)
```

Comment:
This makes no sense:

```go
require.NotNil(t, fetched.Data.Relationships.Workitems.Links)
require.NotNil(t, *fetched.Data.Relationships.Workitems.Links)
```

Once the first line (`require.NotNil(t, fetched.Data.Relationships.Workitems.Links)`) passes you can access the values of `Links`.